### PR TITLE
Added status check to remote builds.

### PR DESCRIPTION
**Pull request by wmarshall484**

### CODE REVIEW

**Comment by wmarshall484** [comment]
Fixes #671

**Comment by ddebrunner** [comment]
Is it possible to share the code with the non-remote contexts, rather than having two copies?

**Comment by ddebrunner** [comment]
Or a package that is common the streams analytic service rest api that can be shared.

**Comment by ddebrunner** [comment]
I forgot I'd added this package to share code for Streaming Analytics service for the contexts:

`src.com.ibm.streamsx.topology.internal.streaminganalytics`

**Comment by wmarshall484** [comment]
I moved the common functionality into `src.com.ibm.streamsx.topology.internal.streaminganalytics.RestUtils`.

**Comment by wmarshall484** [comment]
A complete merging of all duplicated REST code seems outside the scope of #671. I've created issue #701 to track this.

File: java/src/com/ibm/streamsx/topology/internal/streaminganalytics/RestUtils.java [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+        
+        if (!"running".equals(jstring(jsonResponse, "status")))
+            throw new IllegalStateException("Service is not running!");
+    }
+
+    public static String getAPIKey(String userid, String password) throws UnsupportedEncodingException{
+	String api_creds = userid + ":" + password;
+	String apiKey = "Basic "
+	    + DatatypeConverter.printBase64Binary(api_creds
+						  .getBytes("UTF-8"));
```
Comment (ddebrunner) [inline java/src/com/ibm/streamsx/topology/internal/streaminganalytics/RestUtils.java]: Use `StandardCharsets.UTF_8` instead of "UTF-8"

**Pull request merged**